src/test/ui/traits/traits-negative-impls.rs

```diff
@@ -46,7 +46,7 @@ fn dummy2() {
     impl !Send for TestType {}
 
     is_send(Box::new(TestType));
-    //~^ ERROR `dummy2::TestType` cannot be sent between threads safely
+    //~^ ERROR the trait bound `dummy2::TestType: std::marker::Send` is not satisfied
```

Comment:
Can this be changed back?

Reply:
I can switch the logic to make the `rustc_on_unimplemented` text take precedence again. It seemed safer not to, but let me see what I can do to make it clear.

Reply:
done